src/govuk/components/header/template.test.js

```diff
@@ -107,6 +107,20 @@ describe('header', () => {
       expect($firstItem.text()).toContain('Navigation item 1')
     })
 
+    it('renders navigation item with html', () => {
+      const $ = render('header', {
+        navigation: [
+          {
+            href: "#1",
```

Comment:
The tests are failing because the linter's being fussy and wants single quotes here.

```suggestion
            href: '#1',
```